Remotion/Web/Development.WebTesting.UnitTests/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Coypu" version="2.14.0" targetFramework="net45" />

```

Comment:
wieso recht hier, coypu zu referenzieren und nicht auch selenium? normalerweise passiert das autoamtisch beim nuget install-package, dass die abhängigkeiten auch dazu kommen.

Reply:
Das stimmt, wir verwenden jedoch nur `ElementScope` in folgender Testimplementierung:
```csharp
private class TestRequestErrorDetectionStrategy : IRequestErrorDetectionStrategy
{
  public void CheckPageForError (ElementScope scope) => throw new NotImplementedException();
}
```
Da Selenium nicht benötigt wird habe ich es entfernt.

Reply:
Bitte nicht, NuGet mag das garnicht gerne.